docs/user_guide/mtf_linters.rst

```diff
@@ -1,8 +1,70 @@
 Linters
 =================
 
-MTF provides a set of linters for checking containers
+MTF provides a set of linters for checking containers, help files and dockerfiles.
 
 Dockerfile linters
 ~~~~~~~~~~~~~~~~~~
+Dockerfile linters are divided into two python files: `dockerlint.py`_ and `dockerfile_lint.py`_.
+
+.. _dockerlint.py: https://github.com/fedora-modularity/meta-test-family/blob/master/moduleframework/tests/generic/dockerlint.py
+.. _dockerfile_lint.py: https://github.com/fedora-modularity/meta-test-family/blob/master/moduleframework/tests/static/dockerfile_lint.py
+
+`dockerlint.py`_ performs these checks on an existing container image:
+
+* **test_all_nodocs** checks if documentation files shipped by an install RPM packages. There are installed in the ``base image`` or by ``RUN`` commands." . This is only ``WARN`` check.
```

Comment:
I'm only assuming what this option does, so correct me if I'm wrong. How about: ...checks if documentations files shipped by installed RPM packages have been removed. They are usually installed in the ``base image`` and inherited by child layer or installed via the ``RUN`` instruction. ...

Reply:
yes

Reply:
*`documentation files`?